### DISCUSSION TITLE
Superset deck.gl onclick href using javascript

### DISCUSSION THREAD

**Discussion by neers-0901** [apache/superset#27584] [Q&A / Help]
I need to open two URLs from Deck.gl. I am able to open with windows.open. But the thing is I need to pass dynamic paramters based on clicked point. Below is my code and it opens first and second link, not the third. Any help would be greatly appreciated. 

d => "https://latsuperset1.asia.abi.dyn.nesc.nokia.net/superset/dashboard/203/?sitename=" + d.object.extraProps.sitename + "&circle_name=" + d.object.extraProps.circle_name
window.open("https://www.google.com/"); 
e => "https://latsuperset1.asia.abi.dyn.nesc.nokia.net/superset/dashboard/199/,sitename=" + e.object.extraProps.sitename

**Comment by rusackas** [comment]
onClick is a delicate one, as it opens a bunch of security issues. I haven't looked at this in a long time, but I think that's explicitly NOT supported, since, in theory, it could send all sorts of fun things over the wire to any destination. 

As a side note, we'd love to see Nokia on the [In the Wild ](https://github.com/apache/superset/blob/master/RESOURCES/INTHEWILD.md) page if you want to add your org, and feel free to DM me on [slack](http://bit.ly/join-superset-slack) if you ever want to discuss your org's use cases and so forth to get a little closer to the community.
